Example/CustomViewController.swift

```diff
@@ -147,7 +147,7 @@ class CustomViewController: UIViewController {
     // Update the route on the map.
     @objc func rerouted(_ notification: NSNotification) {
         self.navigationMapView.removeWaypoints()
-        self.navigationMapView.show([navigationService.route])
+        self.navigationMapView.updateRouteLine(routeProgress: navigationService.routeProgress, coordinate: navigationService.router.location?.coordinate)
```

Comment:
Use `navigationMapView.updateRouteLine(routeProgress:coordinate:)` instead of `navigationMapView.show(_:legIndex:)` to update the route line during active navigation, when refresh or rerouting event occurs.